.github/workflows/code_scan.yml

```diff
@@ -7,6 +7,26 @@ on:
     - cron: "0 18 * * 1-5"
 
 jobs:
+  Bandit:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
```

Comment:
As same in above context, why using actions/setup-python@v2 instead of actions/setup-python@v3?